src/Microsoft.AspNetCore.Blazor/PlatformInfo.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.AspNetCore.Blazor
+{
+    internal static class PlatformInfo
+    {
+        public static bool IsWebAssembly { get; }
```

Comment:
I know we might eventually have reasons to make this `public`, but it isn't required right now.